src/components/InviteFriendsIcon.js

```diff
@@ -55,14 +54,11 @@ const InviteFriendsIcon = ({ user: { username, numUsersRecruited } }) => {
       >
         <div className={classes.popoverContent}>
           <Typography className={classes.titleSection}>
-            You've recruited{' '}
-            <span style={{ fontWeight: 'bold' }}>
-              {`${numUsersRecruited} friend${isPlural(numUsersRecruited)} `}
-            </span>{' '}
+            helping cats feels even better with friends
```

Comment:
This looks awkward to me. Capital "h", maybe bold title?

![Screen Shot 2021-04-14 at 5 00 12 PM](https://user-images.githubusercontent.com/5924325/114795221-12a09d00-9d5c-11eb-86a4-43a54392a0ae.png)